### PR TITLE
Kot/maria/joinstartandboardgamefunctionality

### PR DESCRIPTION
**Pull request by MariaAT18**

### CODE REVIEW

File: src/main/java/jalau/at18/kingoftokyo/controller/WhoStartRollDiceController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+public class WhoStartRollDiceController {
+    private static final int QUANTITY_DICES = 6;
+    private RollDiceSectionUI rollDiceSectionUI;
+
+    private List<DiceFace> listRandomDiceFace;
+
+    public WhoStartRollDiceController(RollDiceSectionUI rollDiceSectionUI) {
+        this.rollDiceSectionUI = rollDiceSectionUI;
+        rollDiceSectionUI.getRollerDiceButton().addActionListener(e -> rollDices());
```
Comment (AdriOli134) [inline src/main/java/jalau/at18/kingoftokyo/controller/WhoStartRollDiceController.java:19]: Please, Explain me, what is the function of "e ->". I never use it

**Review by AdriOli134 — COMMENTED** [review]
All it is ok, but need refactor for doing the code extensible, for example, in case that there are 3 or more players...with Sarai, we are view a possibility for refactor it. Do rebase, for doing the merge.

**Comment by MariaAT18** [comment]
ok thanks I will do that.

**Comment by MariaAT18** [comment]
The rebase was done

**Review by AdriOli134 — APPROVED** [review]

File: src/main/java/jalau/at18/kingoftokyo/controller/WhoStartRollDiceController.java (hunk 1 above)
Comment (MariaAT18) [reply src/main/java/jalau/at18/kingoftokyo/controller/WhoStartRollDiceController.java:19]: This is a Lambda where for this case we can call the listener.

**Pull request merged**